CHANGELOG.md

```diff
@@ -4,9 +4,30 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+### 1.0.0
+
+**Released**: 2019.09.19
+
+**Commit Delta**: [Change from 0.0.2 release](https://github.com/plus3it/terraform-aws-tardigrade-config-rules/compare/0.0.2...1.0.0)
```

Comment:
Add an entry for 0.0.3 and update this compare link?

Reply:
Fixed